src/Workleap.DomainEventPropagation.Abstractions/ISubscriptionDomainEventBehavior.cs

```diff
@@ -0,0 +1,8 @@
+namespace Workleap.DomainEventPropagation;
+
+public delegate Task SubscriptionDomainEventHandlerDelegate(IDomainEventWrapper domainEventWrapper, IDomainEventSubscriptionContext subscriptionContext, CancellationToken cancellationToken);
```

Comment:
Should we swap context and event parameters? (context, event) seems more logical to me, but maybe it's just me